Snakefile

```diff
@@ -74,7 +74,7 @@ rule germline_cov:
 rule merge_germline:
     """Merge multi-chromosome output from germline_cov into single file"""
     input:
-        expand("{temp_dir}/normal/{chr}.bed", chr=chromosomes, temp_dir=temp_dir)
+        expand("{{temp_dir}}/normal/{chr}.bed", chr=chromosomes)
```

Comment:
What's going on with the double braces here?  I'm just not sure I get these parts properly yet.

Reply:
Snakemake will first run any functions in the inputs before building the DAG and figuring out wildcards. So here the expand() function will run, and it sees {{wildcard}}. Because of the double bracket it doesn't interpret it as an *expansion* wildcard. So the function will return "{temp_dir}/normal/1.bed", "{temp_dir}/normal/2.bed", etc. Then snakemake builds the DAG and fills in the {temp_dir} wildcard itself.

Reply:
This change is required so that we can use a workflow-wildcard temp_dir which is defined in rule gsc_preseg